app_dart/lib/src/service/datastore.dart

```diff
@@ -43,10 +43,13 @@ class DatastoreService {
   /// The [limit] argument specifies the maximum number of commits to retrieve.
   ///
   /// The returned commits will be ordered by most recent [Commit.timestamp].
-  Stream<Commit> queryRecentCommits({int limit = 100, int timestamp}) {
+  Stream<Commit> queryRecentCommits(
```

Comment:
Tests for this?

Reply:
Please refer to the last test in `get_status_test.dart`

Reply:
Doesn't have to be this PR due to the priority, but we should start moving some of the datastore logic we do to be tested in datastore_test.dart. datastore_test.dart doesn't even exist :)